drivers/wifi/esp32/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library_include_directories(${ZEPHYR_BASE}/subsys/net/ip)
```

Comment:
Probably want to add the drivers/modem dir here to make using the base layer easier:
`zephyr_library_include_directories(${ZEPHYR_BASE}/drivers/modem)`